polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -633,6 +652,15 @@ pub mod pallet {
 		MigrateVersionNotifiers,
 		NotifyCurrentTargets(Option<Vec<u8>>),
 		MigrateAndNotifyOldTargets,
+		RemoveExpiredAliasAuthorizations,
+	}
+
+	/// Entry of an authorized aliaser for a local origin. The aliaser `location` is only authorized
+	/// until its inner `expiry` block number.
+	#[derive(Clone, Encode, Decode, Debug, Eq, PartialEq, Ord, PartialOrd, TypeInfo)]
+	pub(crate) struct OriginAliaser {
```

Comment:
I do not really see the reason to make stored types and storage items private. At the end when you need to use it for some reason you just have to copy past them and create an alias storage item.

Reply:
not sure what you mean..

should define this other place? or just make it `pub` instead of `pub(crate)`?